fake-koji/src/main/java/org/fakekoji/jobmanager/project/JDKProjectParser.java

```diff
@@ -285,15 +285,19 @@ void resetPlatform() {
             throw new RuntimeException("Resetting platform error");
         }
 
-        void setPlatform(final String platformId, final String platformProvider) {
+        void setPlatform(final BuildPlatformConfig buildPlatformConfig) {
+            buildPlatform = platformsMap.get(buildPlatformConfig.getId());
```

Comment:
Is this actually correct?
Provider is not part  of id afaik. So platformsMap.get  will return  some  os.arch with set of providers, no? But maybe I mess platform and paltformConfig

Reply:
This applies only to JDKTestProjjects, where build platform does not have a provider